upgrade/upgrade.go

```diff
@@ -109,10 +109,16 @@ func upgrade(currentNodeID, namespace string, config *restclient.Config, lhClien
 				if err = doAPIVersionUpgrade(namespace, config, lhClient); err != nil {
 					return
 				}
+				if err = doCRDUpgrade(namespace, lhClient); err != nil {
```

Comment:
To make the pattern identical, you can rename `CRD` to `CR` for all functions in this package

Reply:
Thank you. Fixed